actions/action-lint/action.yaml

```diff
@@ -0,0 +1,27 @@
+name: "Lint actions and workflows"
+description: "Lint all actions and workflows of a repository"
+inputs:
+  ref:
+    description: "The ref name to checkout the repository."
+    required: false
+    default: ${{ github.ref_name }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Check out repository
+      uses: actions/checkout@v3
+      with:
+        ref: "${{ inputs.ref }}"
+
+    - name: Download "actionlint-matcher.json"
+      run: |
+        wget https://raw.githubusercontent.com/rhysd/actionlint/v1.6.22/.github/actionlint-matcher.json
+        echo "::add-matcher::actionlint-matcher.json"
+      shell: bash
+
+    - name: Check workflow files
+      run: |
+        bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/v1.6.19/scripts/download-actionlint.bash)
```

Comment:
please use the same tag as above. Maybe introduce a param for the tag so its easier to update

Reply:
i changed without parameter. because to make configurable i would have to make an input variable(allowing any user to change also)

or use env var for each step. Wich does not reduce the complexity while updating.

Reply:
You can have a default. I don't think it introduces any complexity

Reply:
@MichaelKora